docs/source/install.rst

```diff
@@ -98,47 +98,71 @@ Specific functionality in Dask may require additional optional dependencies.
+| IPython         | >=7.9.0    | Graph visualization using the cytoscape engine                                                         |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| jinja2          | >=2.10.3   | HTML representations of Dask objects in Jupyter notebooks                                              |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| lz4             | >=4.3.2    | Transparent use of lz4 compression algorithm?                                                          |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| matplotlib      | >=3.1.1    | Color map support for graph visualization                                                              |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| mimesis         | >=5.3.0    | Random bag data generation with ``dask.datasets.make_people()``                                        |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| mmh3            | >=2.4      | Use of murmurhash algorithm for array hashing                                                          |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| numpy           | >=1.21     | Required for dask.array                                                                                |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| pandas          | >=1.3      | Required for dask.dataframe                                                                            |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| psutil          | >=5.7.0    | Factor CPU affinity into CPU count, intelligently infer blocksize for ``dask.dataframe.read_pandas()`` |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| pyarrow         | >=7.0      | Support for Apache Arrow datatypes & engine when storing/reading Apache ORC or Parquet files           |
++-----------------+------------+--------------------------------------------------------------------------------------------------------+
+| python-graphviz | >=0.8.4    | Graph visualization using the graphviz engine                                                          |
```

Comment:
This is tricky because we require both the system graphviz and Python bindings to be installed. The Python bindings are actually just called `graphviz` (https://pypi.org/project/graphviz/), but the `python-graphviz` package on conda-forge is a metapackage containing both the graphviz system package + Python bindings. 

I'm inclined to just call this `graphviz` in this table, but agree it's not ideal

Reply:
That's fine by me, was going to suggest maybe adding links for each library to a relevant GitHub page so that it's a little more clear what pip/conda installation looks like